recipes/themysto/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# this is a conda-forge recipe. See https://github.com/conda-forge for details
+
+{% set version = "0.0.4" %}
+
+package:
+  name: themysto
+  version: {{ version }}
+
+source:
+  # also taken from nb_conda
+  git_url: https://github.com/jcb91/IPython-notebook-extensions.git
+  git_tag: conda
```

Comment:
> Could we please download a tarball?

I don't see why not, although I'm not sure of the path spec for that off the top of my head

> Also, is it alright if we use the original repo to get these ( https://github.com/ipython-contrib/IPython-notebook-extensions )?

So far, the main repo doesn't have an install process that works correctly with conda. I'm attempting to get this working in my fork as a preliminary to fixing it in the source repo. Once (if) that's all merged, yes, it will make sense to switch to the main repo url.

> Also, a nicer tag that matches some version would be nice.

Absolutley, once I've got it building without errors, I'll add a nicer version tag.